Sources/Shared/Extensions/WKWebViewExtensions.swift

```diff
@@ -82,11 +83,37 @@ public extension WKWebView {
       evaluateSafeJavaScript(
         functionName: functionName,
         args: args,
+        frame: frame,
         contentWorld: contentWorld,
         escapeArgs: escapeArgs,
         asFunction: asFunction) { value, error in
           continuation.resume(returning: (value, error))
-      }
+        }
+    }
+  }
+  
+  @discardableResult
+  @MainActor func evaluateSafeJavaScriptThrowing(
```

Comment:
Why not just make the original code throwing instead of a new function?
Also, prefer `let (result, error) = await evaluateSafeJavascript(....)` instead of doing `let error = result.1`

Reply:
I tried this at first but required quite a few changes to existing places in code which ignore the error. 
The only easy way for me to do it was to put a try? or do an empty catch as I couldn't know how to deal with this error everywhere this function is used. I was afraid of doing this because surely adding a try? or an empty catch would raise some eyebrows (even though technically that's what is happening now). I thought this is a better approach because we can chose to deprecate the original method at any time.

And thank you for the suggestion for the `let (result, error)` I always forget this is possible.